scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -97,7 +97,7 @@ object ScalafmtRunner {
       known.find(_.value eq dialect).map(_.source)
 
     def getUnknownError = {
-      val knownStr = known.map(_.source).mkString(",")
+      val knownStr = known.map(_.source.toLowerCase).mkString(",")
```

Comment:
Are you about the ordering of runner dialects? 
It's quite natural to have an order of runner dialects for the Scala first and then for SBT, I suppose. Just as in docs.

Reply:
Or if you are convinced that it's mindless, I doubtless will fix it :)

Reply:
you are worried about the presentation of dialects in some error message. therefore, take care of that and only that, don't change the internal list of dialects.

the reason is that next time someone adds a dialect, he will mess it up for you.

so you can lowercase it here, you can apply sorting etc. but don't change `known`.

Reply:
Ok, now it's only lowercase here.

Reply:
thank you. don't want to sort?

Reply:
Just because it seems a bit strange to me to sort the static collection every time or have a copy of it with sorted values just for the error message. Although your point is quite clear, someone actually can break a sorting by adding a new dialect.

Reply:
well, in that case, a better option is this:
```
    private[ScalafmtRunner] val known = Seq[WithName](
      Scala211,
      scala212,
      Scala212Source3,
      scala213,
      Scala213Source3,
      Sbt0137,
      Sbt1,
      scala3
    ).map(x => withName(x.source.toLowerCase, x.value)).sortBy(_.source)
```

it will also affect the error message which is produced if an incorrect dialect is specified in the configuration. and you don't need to change `getUnknownError`.

Reply:
If I'm correct this sorting puts SBT's dialects at the head. Is it right?

Reply:
you're correct. but it doesn't matter, it will help people do a mental binary search rather than a linear scan.

Reply:
Hmm, if we have the `Seq[WithName]` then calling the `withName` is redundant? Am I missing some things? Anyway, compilation succeeded without it.

Reply:
perhaps i misunderstood what you want to do in the first place. `withName` was taking a lowercase version of the name. did you NOT care about the lowercase name?

Reply:
Wow, shame on me. I'm half asleep. You're 100% right!

Reply:
Hope, now all is ready to merge these two lines 😄